tests/components/device_tracker/test_asuswrt.py

```diff
@@ -1,14 +1,34 @@
 """The tests for the ASUSWRT device tracker platform."""
-
+# pylint: disable=invalid-name
```

Comment:
What are we disabling exactly? If possible can you move the pylint disable to the specific line(s), because if it's here at the top of the file it will apply to everything in it.